components/add-member-form.js

```diff
@@ -0,0 +1,49 @@
+import React from 'react'
+import { Formik, Field, Form, ErrorMessage } from 'formik'
+
+function Button ({ type, disabled }) {
```

Comment:
I'd like to reconcile this with the existing `Button` component. It could happen in this pr or a follow up.

Reply:
Ah yeah I couldn't decide if these should be different buttons. I couldn't get the disabled behavior, I just colored the button differently. You can see it in my PR

Reply:
let's deal with this in a follow up pr.